### DISCUSSION TITLE
How to enable the content like this?

### DISCUSSION THREAD

**Discussion by mendax1234** [pradyunsg/furo#467] [Questions] [answered]
### What's happening?

![image](https://user-images.githubusercontent.com/62934294/176115804-992c6cbd-34b3-4970-9f43-3a61d5f4e0c0.png)
l just want to enable content like this. What should l do?

### Reproducer

1. Build Sphinx documentation at '...' (include build instructions)
2. Go to '...'
3. Click on '...'
4. Scroll down to '...'
5. See wrong thing.

### Expectation

![image](https://user-images.githubusercontent.com/62934294/176115849-b44a8adb-ed4d-485f-aaeb-53d4e6a4efd3.png)
Enable content like this one/

### Code of Conduct

- [X] I agree to follow the [Code of Conduct](https://github.com/pradyunsg/furo/blob/main/CODE_OF_CONDUCT.md).

**Comment by pradyunsg** [comment, marked as answer]
@mendax1234 Thanks, but this is *not* supposed to be filed as an issue. Users seeking support on this project should file a discussion. I've moved this and your other issue into separate discussions. Please don't do this again.

You've also ignored the issue template anyway and not included any reproduction instructions, so I'm not sure what you're looking for.

My guess is that the screen you're viewing this on is not wide enough.

**Comment by mendax1234** [comment]
l mean l want to have the content style in the picture. Since my content now is like this.
![image](https://user-images.githubusercontent.com/62934294/176125472-7d071a17-fd27-47f4-82ab-f6b58fa43133.png)
Thanks!

**Reply by pradyunsg** [reply]
Add headings to the page.

**Reply by mendax1234** [reply]
> Add headings to the page.

How to do that?

![image](https://user-images.githubusercontent.com/62934294/176127799-3ffe79d8-7a0c-4875-8f9a-5bf6186e7100.png)
l mean l don't want this style of content since l see you have another style of content in your project's README page. So, l wonder whether l can have that style of content

**Reply by pradyunsg** [reply]
I'm not sure what you mean by style of content -- I'm guessing you want level 2 headings?

**Reply by mendax1234** [reply]
> I'm not sure what you mean by style of content -- I'm guessing you want level 2 headings?

Sorry for that. 
In the below image, the content is from the right side.
![image](https://user-images.githubusercontent.com/62934294/176130594-545ac7af-be88-4eca-abd0-96cfbb7a7200.png)

But in your default image in your README.
l find that the content is always in the right side. l guess l don't need to click a button to show the content. And l wonder whether the method to see the content is still available now. If so , what should l do to get this function work.
![image](https://user-images.githubusercontent.com/62934294/176130834-6ade5ef6-f76e-4893-8d95-7aac5108771c.png)

**Reply by pradyunsg** [reply]
You'll need a screen that is wide-enough for both the sidebars and the content to fit. You can try zooming out on the browser to see how that looks.

(and, no, Furo does not allow tweaking any of the widths or breakpoints)

**Reply by mendax1234** [reply]
> You'll need a screen that is wide-enough for both the sidebars and the content to fit. You can try zooming out on the browser to see how that looks.
> 
> (and, no, Furo does not allow tweaking any of the widths or breakpoints)

That's it! It works now .Thanks!
